flake.nix

```diff
@@ -4,31 +4,18 @@
   inputs = {
     nixpkgs = { url = "nixpkgs/nixos-unstable"; };
     poetry2nix = { url = "github:nix-community/poetry2nix"; };
+    flake-utils = { url = "github:numtide/flake-utils"; };
```

Comment:
please don't use flake-utils. flake is unstable enough to use it I would prefer to keep things lean even if that means more lines.